src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,10 @@
+#headerContainer{
+	width: 100%;
+	height: 100px;
+	background-color: black;
+	color: white;
+}
+
+#headerTitle{
+	text-align: right;
+}
```

Comment:
you can do nested css here:

`
#headerContainer {
...
   #headerTitle {
     ... }
}
`